### DISCUSSION TITLE
Timeline for 1.7 and community managed SBCs?

### DISCUSSION THREAD

**Discussion by Misza13** [siderolabs/talos#8114] [General]
Hi. Apologies if this is answered somewhere else, but is there a timeline, even a noncommittal one, for 1.7 release?

I am particularly interested in community managed SBCs. I hope to get my TuringPi RK1 modules Q1 next year, and I was interested in trying out Talos on them. I'm interested in timeline for the aforementioned feature to make plans for either using Talos, its [RK3588 fork](https://github.com/nberlee/talos) or look for something else entirely.

Thanks!

**Comment by smira** [comment]
The discussion is happening in #8065, we haven't committed to any timeline for it yet.

**Reply by ringods** [reply]
SBC = Single Board Computer? Please write out acronyms once in the beginning of a message before using the acronym.
